assets/javascripts/discourse/initializers/qa-edits.js

```diff
@@ -186,7 +186,7 @@ function initPlugin(api) {
     const result = [];
     const model = helper.getModel();
 
-    if (model.topic.is_qa) {
+    if (model.topic?.is_qa) {
```

Comment:
Can we add a test case for this here because I'm not sure I understand the fix here. Is there a situation where a post does not have a topic attribute and is still being rendered?